src/Raven.Client/Documents/Operations/Replication/PullReplicationAsSink.cs

```diff
@@ -65,6 +66,9 @@ public override DynamicJsonValue ToJson()
             if (string.IsNullOrEmpty(HubName))
                 throw new ArgumentException("Must be not empty", nameof(HubName));
 
+            if (string.IsNullOrEmpty(Url))
+                throw new ArgumentException("Must be not empty", nameof(Url));
```

Comment:
what will happen if I created PullReplicationAsSink in prev version without Url and now I want to edit it e.g. via Studio? Where and when '.ToJson' is used? shouldn't you assert this in a ctor?

Reply:
done